patches/@ecl/ec-component-footer+1.1.0.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/node_modules/@ecl/ec-component-footer/ec-component-footer.twig b/node_modules/@ecl/ec-component-footer/ec-component-footer.twig
```

Comment:
Did we open a PR or a ticket in ECL to fix this on their end?

Reply:
Created.